bin/hubot

```diff
@@ -7,6 +7,7 @@ Hubot    = require '..'
 Fs       = require 'fs'
 OptParse = require 'optparse'
 Path     = require 'path'
+env      = require('dotenv').config({silent: true}); 
```

Comment:
you don’t need to set the `env` variable. I would also suggest to move up `require('dotenv').config({silent: true});` to the very first code line, as suggested in https://www.npmjs.com/package/dotenv#usage